tests/unit/conftest.py

```diff
@@ -413,7 +413,22 @@ def default_factory(default_project):
 
 
 @pytest.fixture()
-def lifecycle_service(default_project, default_factory, tmp_path):
+def default_build_plan():
+    from craft_application.models import BuildInfo
+    from craft_providers import bases
+
+    return [
+        BuildInfo(
+            platform="amd64",
```

Comment:
mind calling this generic-x86-64 or something like that so no one gets confused with the intent of the field?